src/styles.css

```diff
@@ -0,0 +1,27 @@
+@import "./styles/bloks/card.css";
+@import "./styles/bloks/stars.css";
+
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+  font-style: normal;
+  font-weight: 500;
+}
+
```

Comment:
maybe it's would be great to fetch 400 font as well? I see you didn't set 500/700 for all typography, it means that it's using default 400